plugins/main/public/components/common/data-source/pattern/pattern-data-source-repository.ts

```diff
@@ -0,0 +1,128 @@
+import { tDataSource } from "../data-source";
+import { DataSourceRepository } from "../data-source-repository";
+import { PatternDataSource } from "./pattern-data-source";
+import { GenericRequest } from '../../../../react-services/generic-request';
+import { AppState } from '../../../../react-services';
+
+/*
```

Comment:
Is this necessary or can it be deleted?

Reply:
Removed in 6583f4da21c0fc54d8fd3f0cbe98e79e52de83d9